modules/nf-core/ensemblvep/download/main.nf

```diff
@@ -2,10 +2,10 @@ process ENSEMBLVEP_DOWNLOAD {
     tag "$meta.id"
     label 'process_medium'
 
-    conda "bioconda::ensembl-vep=108.2"
+    conda "bioconda::ensembl-vep=110.0"
```

Comment:
weird, I thought this was already fixed in the modules update PR. Maybe merge in dev?